tests/core/test_search.py

```diff
@@ -315,7 +314,7 @@ def test_create_vespa_search_params(
             20,
             10,
             0,
-            "ABC",
+            ["ABC"],
```

Comment:
Does this affect the front-end?

Reply:
It does not, I made sure last week that `continuation_tokens` are not currently being used there